config/locales/en/instance_user_role_requests.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  instance_user_role_requests:
+    new:
+      header: 'Create Role Request'
+      hint: 'Please provide the following information for us to review your request.'
+    create:
+      success: 'Your role request was submitted.'
```

Comment:
'Your request for more permissions has been submitted.'

Do they get a notification if they're approved?